incubator/java-microprofile/README.md

```diff
@@ -1,47 +1,63 @@
-# Java Microprofile
+# Java Microprofile Stack
 
-A stack is the foundation of your application which provides a prototype platform definition for Java Microprifile based projects.
+The Java Microprofile stack provides a consistent way of developing microservices based upon the [Eclipse MicroProfile® specifications](https://microprofile.io). This stack lets you use [Maven](https://maven.apache.org) to develop applications for [Open Liberty](https://openliberty.io) runtime, which includes OpenJDK with container-optimizations in OpenJ9.
 
-The Java Microprofile stack uses a parent POM to manage dependency versions and provide required capabilities and plugins whilst using Maven. Specifically, this stack enables adoptopenjdk/openjdk8-openj9 and Open Liberty version `19.0.0.5`.
+The Java Microprofile stack uses a parent Maven project object model (POM) to manage dependency versions and provide required capabilities and plugins.
+
+This stack is based on `adoptopenjdk/openjdk8-openj9` and `Open Liberty v19.0.0.5`. It provides live reloading during development by utilizing `loose application` capabilities.
```

Comment:
what do we mean by `adoptopenjdk/openjdk8-openj9`?

Reply:
We use OpenJDK v8 binaries built by AdoptOpenJDK with open j9 optimisations..  I was trying to make that clear.

Reply:
replaced with the English equivalent